CONTRIBUTING.md

```diff
@@ -2,14 +2,15 @@
 
 ## Add a new blog post
 
-* Create a markdown file and add your blog content along with the metadata in    the following format
+* Create a markdown file and add your blog content along with the metadata in the following format
 ```
 ---
 title: Title of the blog
 author: Author name
 author_info: Author information
-tags: All the tags with space in between (Example: openebs localpv)
+tags: All the tags with comma separation in between (Example: openebs, localpv)
 date: Date of publication in dd-mm-yyyy format
+excerpt: Short description about the blog in about 200 hundred words
```

Comment:
Short description about the blog in about 200 hundred words. Words beyond 200 range will be clipped.